lib/board.rb

```diff
@@ -1,5 +1,7 @@
 class Board
   attr_reader :board
+  attr_reader :POSITION_MIN
+  attr_reader :POSITION_MAX
```

Comment:
For information, in Ruby you'll sometimes see a `private` section towards the bottom of a class. E.g:
`private
attr_reader :board`.  

This means that you can initialise `@board` in `def initialize` but there after, you do not need to reference the variable with the `@` prefix. I can show you an example if it is not clear what I mean - let me know. 

Also, you may already know, but just in case you don't, any methods also declared after the `private` word, will also be assumed private - an indication that they should not be used outside of this class. 

You have used private sections in some classes but not others. Consistency is a good habit to get into.